.changeset/cool-falcons-develop.md

```diff
@@ -0,0 +1,8 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+- To match native, Radio Group now sets its `value` to that of its checked radio even if that radio is disabled.
+- To match native, Radio Group now sets its `value` to that of the last checked radio instead of the first.
+- When `value` is set initially and a radio is checked, Radio Group now defers to the checked radio and overwrites its initial `value`.
+- When `value` is set initially and an option is selected, Dropdown now defers to the checked radio and overwrites its initial `value`.
```

Comment:
A subtlety that we haven't been accounting for in component. Checkbox Group will also need this change. On my list to do next.

Reply:
Checkbox Group also doesn't support setting `value` initially.